shakenfist/blob.py

```diff
@@ -97,10 +98,22 @@ def locations(self):
             return []
         return locs.get('locations', [])
 
+    @locations.setter
+    def locations(self, new_locations):
+        self._db_set_attribute('locations', {'locations': new_locations})
+
     @property
     def info(self):
         return self._db_get_attribute('info')
 
+    @property
+    def ref_count(self):
+        '''Counts artifact references to the blob'''
```

Comment:
Just betweent the two of us, PEP8 says this should be triple double quotes, so:

`"""Counts the things."""`